addons/website/static/src/scss/website.scss

```diff
@@ -1323,6 +1323,15 @@ header {
                     }
                 }
             }
+            .o_header_mobile_buttons_wrap, .btn[data-bs-toggle="offcanvas"] {
+                color: inherit;
+            }
+            .navbar-toggler-icon {
+                background-color: currentColor;
+                background-image: none;
+                mask: $navbar-light-toggler-icon-bg;
+                -webkit-mask: $navbar-light-toggler-icon-bg;
+            }
```

Comment:
> What prevents you from doing this starting from 17.0 ? I don't see anything that could break stable 🙂

I don'want to change existing design. for instance if a customer has a header with red text and a white "hamburger" button. I don't want its button to turn red.

Reply:
> But wasn't it mentioned in the task that we don't care about breaking that ? 😮 It would still be changed when the user will upgrade anyway 🤷

In task it's said : "I can't imagine someone actually wanting all texts to be white but did want the toggler to be black"
In this case, it was about fixing something: text dark and hamburger light for instance...
My opinion is that using dark or light hamburger icon is not a good solution. If the text color is red (+- 50% intensity) it sould work with dark or light image behind it but having to choose between a light or in dark hamburger in this case is impossible ? How to kown the intensity of the image behind the header...

Reply:
@qsm-odoo Could you confirm it's ok that this PR is for master

Reply:
@bvr-odoo No, what I was explaining in the task is that we can ignore the stable policy for this one. Our text color option is supposed to allow any transparent header to work (at least over a kinda-unifiee dark/light color/image), I feel like we improve the life our users more by doing this in 17.0 (which will be used for 3 years at least) than in master. And the risk is negligible: at worse, a black or white toggler will change color... to use a color that is chosen by the user for the neighbor text, so it will probably work most of the time. Sure, you could break something (the background behind can be white on the left and black on the right... in which case, it's probably already broken anyway)... but in practice, I feel the "value" is greater than that "risk". If the "bug" was in 15.0 too, I'd say the same: let's target 17.0, because no point in taking that "risk" in a version that users used for 3 years and where probably less and less customer **start** a new database. But **new 17.0 users** will continue to be a thing for a few years (our partners do not use saas versions normally and they probably won't adopt 18.0 immediately for all their use cases, who knows).